src/airflow_tools/filesystems/impl/blob_storage_filesystem.py

```diff
@@ -30,20 +30,35 @@ def write(self, data: str | bytes | BytesIO, path: str):
         )
         self.hook.upload(container_name=container_name, blob_name=blob_name, data=data)
 
+    def delete_file(self, path: str):
+        container_name, blob_name = _get_container_and_blob_name(path)
+        self.hook.delete_file(
+            container_name, blob_name, is_prefix=False, ignore_if_missing=True
+        )
+
+    def create_prefix(self, prefix: str):
+        # What to do here?
```

Comment:
Can we write the zero bytes data as an object with a trailing slash? Not sure by I think that may be what the S3 UI does, when you create an empty "directory" maybe it works here too:

```python
        container_name, blob_name = _get_container_and_blob_name(f'{prefix}/')
        self.hook.upload(container_name, blob_name, data=b"")
```